qemu/target-arm/unicorn_arm.c

```diff
@@ -9,8 +9,9 @@
 #include "unicorn_common.h"
 #include "uc_priv.h"
 
-
+#ifndef TARGET_WORDS_BIGENDIAN
```

Comment:
then Mips should fix this issue, too.

for duplicated definition, you can put the related variables into qemu/header_gen.py, then run "make header" to generate new header files, and the problem will be fixed.